feature/s3/transfermanager/api_op_PutObject_integ_test.go

```diff
@@ -1,7 +1,7 @@
 //go:build integration
 // +build integration
 
-package s3transfermanager
+package transfermanager
```

Comment:
is the rename required or just something we prefer?

Reply:
required, this package is `transfermanager`